distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/CatalogBundle.java

```diff
@@ -176,4 +176,27 @@ public void setupCaching(boolean cachingEnabled) throws IOException {
         Configuration configuration = adminConfig.getConfiguration(RESOURCE_DOWNLOAD_MANAGER_PID, null);
         configuration.update(updatedProperties);
     }
+
+    public void setDownloadRetryDelay(int delay) throws IOException
+    {
+        setConfigProperty(RESOURCE_DOWNLOAD_MANAGER_PID, "delayBetweenAttempts", delay);
+    }
+
+    private void setConfigProperty(String pid, String propertyName,
```

Comment:
If I remember correctly, this class already had a couple methods setting config properties. We should be able to refactor those methods to use this one.